src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcTokenHandler.php

```diff
@@ -44,7 +41,6 @@ public function __construct(
         private string $audience,
         private array $issuers,
         private string $claim = 'sub',
-        private ?LoggerInterface $logger = null,
         private ClockInterface $clock = new Clock()
```

Comment:
Let's add a small BC layer because this will break the bundle v6.3 otherwise:
```suggestion
        private ClockInterface|LoggerInterface|null $clock = new Clock()
```
then trigger a deprecation 
and add a line in the changelog

Reply:
done